oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/process/ServiceLabelRecord.java

```diff
@@ -0,0 +1,126 @@
+import org.apache.skywalking.oap.server.core.storage.type.StorageBuilder;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+import static org.apache.skywalking.oap.server.core.source.DefaultScopeDefine.SERVICE_LABEL;
+
+/**
+ * Process have multiple labels, such as tag.
+ * {@link ServiceLabelRecord} could combine them in the service level.
+ * It could help to quickly locate the similar process by the service and label.
+ */
+@Setter
+@Getter
+@Stream(name = ServiceLabelRecord.INDEX_NAME, scopeId = SERVICE_LABEL,
+        builder = ServiceLabelRecord.Builder.class, processor = MetricsStreamProcessor.class)
+@MetricsExtension(supportDownSampling = false, supportUpdate = false)
+@EqualsAndHashCode(of = {
+        "serviceId",
+        "label"
+})
+public class ServiceLabelRecord extends Metrics {
+
+    public static final String INDEX_NAME = "service_label";
+    public static final String SERVICE_ID = "service_id";
+    public static final String LABEL = "label";
+
+    @Column(columnName = SERVICE_ID)
+    private String serviceId;
+    @Column(columnName = LABEL, length = 50)
+    private String label;
+
+    @Override
+    public boolean combine(Metrics metrics) {
+        return true;
+    }
+
+    @Override
+    public void calculate() {
+    }
+
+    @Override
+    public Metrics toHour() {
+        return null;
+    }
+
+    @Override
+    public Metrics toDay() {
+        return null;
+    }
+
+    @Override
+    protected String id0() {
+        return this.serviceId + Const.ID_CONNECTOR + new String(Base64.getEncoder()
+                .encode(label.getBytes(StandardCharsets.UTF_8)), StandardCharsets.UTF_8);
+    }
+
+    @Override
+    public void deserialize(RemoteData remoteData) {
+        setServiceId(remoteData.getDataStrings(0));
+        setLabel(remoteData.getDataStrings(1));
+        setTimeBucket(remoteData.getDataLongs(0));
+    }
+
+    @Override
+    public RemoteData.Builder serialize() {
+        final RemoteData.Builder builder = RemoteData.newBuilder();
+        builder.addDataStrings(serviceId);
+        builder.addDataStrings(label);
+        builder.addDataLongs(getTimeBucket());
+        return builder;
+    }
+
+    @Override
+    public int remoteHashCode() {
+        return this.hashCode();
+    }
+
+    public static class Builder implements StorageBuilder<ServiceLabelRecord> {
+
+        @Override
+        public ServiceLabelRecord storage2Entity(Convert2Entity converter) {
+            final ServiceLabelRecord record = new ServiceLabelRecord();
+            record.setServiceId((String) converter.get(SERVICE_ID));
+            record.setLabel((String) converter.get(LABEL));
+            return record;
+        }
+
+        @Override
+        public void entity2Storage(ServiceLabelRecord entity, Convert2Storage converter) {
+            converter.accept(SERVICE_ID, entity.getServiceId());
+            converter.accept(LABEL, entity.getLabel());
```

Comment:
No time bucket to the database, I assume this makes TTL fails. Was mentioned in https://github.com/apache/skywalking/pull/8840#discussion_r846746815

Reply:
Add the `time_bucket` to the column.